src/main/java/seedu/savenus/ui/MainWindow.java

```diff
@@ -273,15 +272,18 @@ private CommandResult executeCommand(String commandText) throws CommandException
                     || commandResult.getFeedbackToUser().equals(InfoCommand.BUDGET_INFO)
```

Comment:
@robytanama maybe consider finding a way to modularise this check? Many repetitive OR statements which could be avoided.